pkg/install/stack/usercluster-mla/stack.go

```diff
@@ -0,0 +1,388 @@
+/*
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package userclustermla
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+
+	"k8c.io/kubermatic/v2/pkg/controller/seed-controller-manager/mla"
+	"k8c.io/kubermatic/v2/pkg/install/helm"
+	"k8c.io/kubermatic/v2/pkg/install/stack"
+	"k8c.io/kubermatic/v2/pkg/install/util"
+	"k8c.io/kubermatic/v2/pkg/log"
+
+	corev1 "k8s.io/api/core/v1"
+	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	UserClusterMlaNamespace    = "mla"
+	UserClusterMlaChartsPrefix = "mla"
+
+	MlaSecretsChartName   = "mla-secrets"
+	MlaSecretsReleaseName = MlaSecretsChartName
+	MlaSecretsNamespace   = UserClusterMlaNamespace
+
+	AlertmanagerProxyChartName   = "alertmanager-proxy"
+	AlertmanagerProxyReleaseName = AlertmanagerProxyChartName
+	AlertmanagerProxyNamespace   = UserClusterMlaNamespace
+
+	ConsulChartName   = "consul"
+	ConsulReleaseName = ConsulChartName
+	ConsulNamespace   = UserClusterMlaNamespace
+
+	CortexChartName   = "cortex"
+	CortexReleaseName = CortexChartName
+	CortexNamespace   = UserClusterMlaNamespace
+
+	GrafanaChartName   = "grafana"
+	GrafanaReleaseName = GrafanaChartName
+	GrafanaNamespace   = UserClusterMlaNamespace
+
+	LokiChartName   = "loki-distributed"
+	LokiReleaseName = LokiChartName
+	LokiNamespace   = UserClusterMlaNamespace
+
+	MinioChartName   = "minio"
+	MinioReleaseName = MinioChartName
+	MinioNamespace   = UserClusterMlaNamespace
+
+	MinioLifecycleMgrChartName   = "minio-lifecycle-mgr"
+	MinioLifecycleMgrReleaseName = MinioLifecycleMgrChartName
+	MinioLifecycleMgrNamespace   = UserClusterMlaNamespace
+
+	MlaIapChartName   = "iap"
+	MlaIapReleaseName = MlaIapChartName
+	MlaIapNamespace   = UserClusterMlaNamespace
+)
+
+type UserClusterMLA struct{}
```

Comment:
Please settle on either calling it "Mla" or "MLA". In https://github.com/kubermatic/kubermatic/blob/master/pkg/apis/kubermatic/v1/mla_admin_settings.go we use "MLA" (all caps).

Reply:
adjusted it to `MLA` in [6b5c482](https://github.com/kubermatic/kubermatic/pull/11008/commits/6b5c4828a1efecbd3a52e68aa04cc152f2721e4c)